src/libYARP_dev/include/yarp/dev/IPidControl.h

```diff
@@ -16,6 +17,14 @@ namespace yarp
     {
         class IPidControlRaw;
         class IPidControl;
+
+        enum YARP_dev_API PidControlTypeEnum
```

Comment:
@randaz81 
do you mind if I add a couple of functions on top of this enum?
To be used in std containers I need to have a comparison function and an hash function.